callstats-java-sdk/src/test/java/io/callstats/sdk/CallStatsTest.java

```diff
@@ -0,0 +1,73 @@
+package io.callstats.sdk;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+import org.junit.Test;
+
+/**
+*
+* @author Karthik Budigere
+*/
+public class CallStatsTest{
+	CallStats callstatslib;
+	CallStatsInitListener listener;
+	
+	@Test
+	public void intializeTest() {
+		callstatslib = new CallStats();
+		listener = mock(CallStatsInitListener.class);
+		callstatslib.intialize(175240362, "Gvd126EUWQheaWQX9mwmeWIbzvs=", "jit.si.345",listener);
+		try {
+			Thread.sleep(1000);
+		} catch (InterruptedException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+		String msg = "SDK authentication successful";
+		verify(listener).onInitialized(msg);
+	}
+	
+	@Test
+	public void intializeWithInvalidAppIdTest() {
+		callstatslib = new CallStats();
```

Comment:
You could use annotation expectedException = xxxx, instead of checking instance type